baybe/utils/dataframe.py

```diff
@@ -66,19 +66,19 @@ def to_tensor(*x: np.ndarray | pd.DataFrame) -> Tensor | tuple[Tensor, ...]:
     return out
 
 
-def add_fake_results(
+def add_fake_measurements(
     data: pd.DataFrame,
     targets: Collection[Target],
     good_reference_values: dict[str, list] | None = None,
     good_intervals: dict[str, tuple[float, float]] | None = None,
     bad_intervals: dict[str, tuple[float, float]] | None = None,
-) -> None:
+) -> pd.DataFrame:
     """Add fake results to a dataframe which was the result of a BayBE recommendation.
```

Comment:
Shouldn't this then also be "Add fake measurements" if the function itself was renamed?

Reply:
changed (had to delete `BayBE` to fit into line length limit)